documentation/API_documentation/ConnectivityInsights-User-Story.md

```diff
@@ -0,0 +1,10 @@
+# Connectivity Insights API User Story
+
+| **Item** | **Details** |
+| ---- | ------- |
+| ***Summary*** | As an application developer i need to be able to configure what my application requires in terms of network perfomance metrics and be able to get visibility into actual network performance as experienced by my customers in comparison to the expected thresholds configured. In addtion i would also like to get notifications when the network is not able to meet the thresholds configured for indivial customers.  |
```

Comment:
I think 'configure' may be misinterpreted , as it makes it sound like an programming action that chnages network treatment (like QoD). 

As an application developer I need to be able to understand what my application requires in terms of network performance demands,  and be able to determine the likelihood that the network will be able to deliver performance that meets those demands.  I can represent my application's network performance demands as a set of metrics, and register these with the network. At runtime I can query the network to determine if a given customer device is likely to receive that level of network performance across each metric.  In addition I would also like to get notifications when the network is not able to meet the network performance thresholds for given customers.